src/interceptors/fetch/index.ts

```diff
@@ -46,13 +47,21 @@ export class FetchInterceptor extends Interceptor<HttpRequestEventMap> {
 
       this.logger.info('awaiting for the mocked response...')
 
+      const signal = interactiveRequest.signal
```

Comment:
Will the original request's abort controller propagate to the interactive request instance too? 

I'm wondering if we should account for `request.signal` directly in case the user aborts it instead of aborting a request in the `request` listener (where they have access to the interactive request). 

I will take a closer look at the tests, I think you've answered this question in there.

Reply:
Okay, got it now. There's no way to abort the request from the request listener because you have to do that via the abort controller you set on the original request. 

I wonder if this isn't a design flaw on our part since allowing the user to abort a request in the request listener may be a good feature to have. Regardless, irrelevant in the context of this fix.

Reply:
Yes because on [line 63](https://github.com/mswjs/interceptors/pull/394/files/e0ca09dc464e815ef00f9ccc292cd523774c1734#diff-4593f50c24857397e849d8d6618757914a847fa2642a22bfaa3b703e15cb4969R63) we `Promise.race` the resolution and the rejection.